### PR TITLE
Adjusted code for use at Bouw7

### PR DESCRIPTION
**Pull request by haroldiedema**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
-    "name":              "hostnet/accessor-generator-plugin-lib",
+    "name":              "bouw7/accessor-generator-plugin",
```
Comment (ping-localhost) [inline composer.json:2]: You cannot change the package name. If you wish to do so, please make a fork and adapt this lib to your companies expectations.

**Comment by haroldiedema** [comment]
Oops :D hahahahaha

File: composer.json (hunk 1 above)
Comment (haroldiedema) [reply composer.json:2]: Sorry Mitchell :joy: Ik lag te slapen.

Comment (ping-localhost) [reply composer.json:2]: Goedemorgen dan maar @haroldiedema :coffee:

**Pull request closed without merging**